config/webpack/development.js

```diff
@@ -27,6 +28,19 @@ module.exports = merge(sharedConfig, {
     historyApiFallback: true,
     watchOptions: {
       ignored: /node_modules/
+    },
+    proxy: {
+      '/': {
+        target: `http://${settings.dev_server.host}:${env.PORT || '3000'}`,
```

Comment:
is there a better ENV var name? (so that we can use with webpacker or foreman Procfile)?

Reply:
I'll see what I can find..